### ISSUE TITLE
Syntax highlighting does not cover the entire file

### ISSUE THREAD

**Issue by roycetech** [idleberg/sublime-applescript#10]
I have a fairly large handler I used for resizing windows and I noticed that the syntax highlighting does not cover the entire file.  In addition, those handlers that are not highlighted do not appear in the list when I trigger the goto Symbol. 

A large library is probably a bad idea and should be refactored, but in the meantime, it might be simpler not to have this limitation?

My library is 1004 lines with comments, inline unit tests.

**Comment by idleberg** [comment]
I don't think that this an issue related to this package, but an issue with Sublime Text itself. I'm aware of limitations when it comes to very [long lines](https://github.com/sublimehq/sublime_text/issues/513), but not with long files. Maybe the parser takes a bit longer when it comes to long files?

**Comment by roycetech** [comment]
I'm not really sure because I work on a legacy project with large files (10k lines) but I never encountered this issue. I've never worked on Sublime Text plugin but could try to dig when I get some free time and see if I can find anything. BTW thank you for your plugin, I find it very useful.
